hack/examples/bases/nutanix/clusterclass/kustomization.yaml.tmpl

```diff
@@ -30,12 +30,25 @@ patches:
     - op: "remove"
       path: "/spec/variables"
 
+# Delete the certSANs from the template
+# They will be added by the handler.
 - target:
     kind: KubeadmControlPlaneTemplate
   patch: |-
     - op: "remove"
       path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/certSANs"
 
+# Delete the kube-vip file.
+# Will be templated and added back in the handler.
+- target:
+    kind: KubeadmControlPlaneTemplate
+  patch: |-
+    - op: test
+      path: /spec/template/spec/kubeadmConfigSpec/files/0/path
+      value: "/etc/kubernetes/manifests/kube-vip.yaml"
```

Comment:
could you please elaborate on what does this test op do.

Reply:
https://jsonpatch.com/#test